specification/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker App Specification
+
+This section describes all the requirements for docker-app interoperability.
+
+## YAML Documents
+
+A Docker App is a set of 3 YAML documents:
+* `metadata`
+* `docker-compose`
+* `settings`
+
+These documents can be split in 3 different files or merged into one YAML file, using the [multi document YAML feature](http://yaml.org/spec/1.2/spec.html#id2760395).
```

Comment:
This leaves the order of the documents, and whether that is relevant, as unspecified. Can I put the three docs in an arbitrary order? What is used to detect the individual documents if so.

Reply:
There's an issue about this #249 , but you're right I'll mention it as for now maintaining the order is mandatory.